core/src/main/java/org/codice/imaging/nitf/core/common/impl/NitfInputStreamReader.java

```diff
@@ -125,4 +125,10 @@ public final void skip(final long count) throws NitfFormatException {
             throw new NitfFormatException(GENERIC_READ_ERROR_MESSAGE + ex.getMessage(), numBytesRead);
         }
     }
+
+    @Override public void close() throws Exception {
```

Comment:
`@Override` would be better on a separate line.

Reply:
I vaguely remember us running into a problem with closing this stream because it was being passed around within Alliance. I think that since we've added `.end()` to the fluent classes, we might be ok to use this, now. We should verify that it doesn't cause a problem outside of imaging-nitf, however.